### PR TITLE
Mood

### PR DESCRIPTION
**Pull request by TannerDale**
# Change Log

- Closes #

## What I did:

- Add mood type
- single mood query
- mood creation mutation

## Things to consider:

- MIGRATE + SEED
- no testing yet

## Test coverage

- [] 100% Model Testing
- [] 100% Request Testing

If below 100%, what still needs testing:

- no testing done yet, next thing in line

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,6 +28,10 @@ gem 'graphql'
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 gem 'rack-cors'
 
+group :test do
+  gem 'database_cleaner-active_record'
```
Comment (TannerDale) [inline Gemfile:32]: Helps keep the database data simple and not flow into other tests

File: app/graphql/types/mood_type.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module Types
+  class MoodType < Types::BaseObject
+    field :id, ID, null: false
+    field :mood, Integer, null: false
+    field :description, String, null: true
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+  end
+end
```
Comment (TannerDale) [inline app/graphql/types/mood_type.rb:8]: Mood Data
Can query for:
- id
- mood
- description
- created at

File: app/models/mood.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Mood < ApplicationRecord
+  belongs_to :user
+
+  validates :mood, presence: true
+end
```
Comment (TannerDale) [inline app/models/mood.rb:5]: Belongs-to has-many with user
mood is required

File: db/seeds.rb [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
 
-(0..10).to_a.each do |i|
-  User.create!(username: "user #{i}", password: 'hi', password_confirmation: 'hi')
+(1..10).to_a.each do |num|
+  Mood.create!(
+    mood: [1, 2, 3, 4, 5].sample,
+    description: "Mood #{num}",
+    user_id: user.id
+  )
 end
```
Comment (TannerDale) [inline db/seeds.rb:13]: Seeds for DB normalization

**Review by nzambonivergara — APPROVED** [review]

**Review by WMudgeEllis — APPROVED** [review]

**Pull request merged**